internal/app/singularity/config_fakeroot_linux.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package singularity
+
+import (
+	"fmt"
+
+	"github.com/sylabs/singularity/internal/pkg/fakeroot"
+)
+
+// FakerootConfigOp defines a type for a fakeroot
+// configuration operation.
+type FakerootConfigOp uint8
+
+const (
+	// FakerootAddUser is the operation to add a user fakeroot mapping
+	FakerootAddUser FakerootConfigOp = iota
+	// FakerootRemoveUser is the operation to remove a user fakeroot mapping
+	FakerootRemoveUser
+	// FakerootEnableUser is the operation to enable a user fakeroot mapping
+	FakerootEnableUser
+	// FakerootDisableUser is the operation to disable a user fakeroot mapping
+	FakerootDisableUser
+)
+
+// FakerootConfig allows to add/remove/enable/disable a user fakeroot
+// mapping entry in /etc/subuid and /etc/subgid files.
+func FakerootConfig(username string, op FakerootConfigOp) error {
+	subUIDConfig, err := fakeroot.GetConfig(fakeroot.SubUIDFile, true, nil)
```

Comment:
Maybe closing a config should be in a defer call?

Reply:
same as above